src/System.Text.Encoding.CodePages/src/System/Text/DecoderNLS.cs

```diff
@@ -37,13 +37,6 @@ internal DecoderNLS(EncodingNLS encoding)
             Reset();
         }
 
-        // This is used by our child deserializers
```

Comment:
Isn't this needed?

Reply:
The type isn't serializable in .NET Core.